packages/conf-lld/conf-lld.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Léo Andrès <contact@ndrs.fr>"
+authors: "The LLVM team"
+homepage: "http://llvm.org"
+bug-reports: "https://llvm.org/bugs/"
+license: "MIT"
+build: [
+  ["wasm-ld" "-version"] { os-distribution != "homebrew" }
```

Comment:
```suggestion
  ["wasm-ld" "-version"]
```

The responsibility (in the sense of MUST) of the package is to check if the expected binary is in the `PATH`. It can also optionally (in the sense of MAY) provide ways to automatically set things up (by way of `depexts` or by more effectful `build` recipe).

I think that in this case we want to remove the guard otherwise the build always succeeds on homebrew even if `wasm-ld` is unavailable.

Reply:
According to https://stackoverflow.com/questions/57322431/wasm-ld-not-found-after-performing-brew-install-wabt#comment139926501_57367163 it looks like it may need the [`lld`](https://formulae.brew.sh/formula/lld) package on homebrew? That also depends on llvm.

Reply:
See: https://github.com/ocaml/opam-repository/pull/27381/files#r1954570873